Transcripts/EP_21_Amir_Slihefendic.md

```diff
@@ -1,38 +1,38 @@
 Amir Salihefendic_Ready
 **Michaela:** [00:00:00] Hello, and welcome to the software engineering unlocked 
-podcast. I'm your host, dr. McKayla. And today I have the pleasure to talk 
-Amir Salihefemdic. Amir is the founder and CEO of todoist. It's just a 
-company behind the diely successful productivity app to do list. And the 
-collaboration platform twist the slogan on a website, says we created tools to 
+podcast. I'm your host, Dr. McKayla and today I have the pleasure to talk to
+Amir Salihefendić. Amir is the founder and CEO of Doist. Doist is the  
+company behind the widely successful productivity app ToDoist and the 
+collaboration platform Twist. The slogan on their website says we create tools to 
 promote a calmer and more balanced, more fulfilling way to work and lift.
 And that's exactly why I'm following the company. And also Amir's work already 
 for years. Not only does he promote a remote work culture, but he also stands 
-behind a more meaningful and self control worklife. But before I welcome, we are 
-one to two invite you to become an early access. Read off some of my draft 
+behind a more meaningful and self control worklife. But before I welcome Amir, I wanted
+to invite you to become an early access reader of some of my draft 
 chapters for my poetry book.
-Sooner, but give my last, could you workshop before my summer break? My next 
-workshops are then in August in the coming month, I will focus completely on 
-writing the code for your book and doing some original research around code 
-reviews. So if you want to read what I write and follow the process, signed up 
-on McKayla giler.com/code you book, and follow me on Twitter.
+Sooner, but give my last, could you workshop before my summer break. My next 
```

Comment:
I dont understand whats being said in this sentence early on
After 'sooner' 
and before 'summer' 
is not clear to me.

Reply:
Sorry, I just saw this now. It looks great in the final pass. Thank you.